dev-util/rtx-cli/rtx-cli-1.29.7.ebuild

```diff
@@ -0,0 +1,326 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Auto-Generated by cargo-ebuild 0.5.4
```

Comment:
"auto-generated" is a strong word... took a bit of massaging to get this working, annoyingly.

I want to write something to keep our ebuilds automatically updated. Will think about the best way to do this, for this ebuild I used `cargo ebuild` to generated the list of `CRATES`, but everything else I had to add myself :|